.pipelines/onebranch/templates/template-buildrp-builddocker.yml

```diff
@@ -0,0 +1,12 @@
+steps:
+- task: onebranch.pipeline.imagebuildinfo@1
+  displayName: Build Multi Stage Dockerfile
+  inputs:
+    repositoryName: aro-rp
+    dockerFileRelPath: ./Dockerfile.aro-multistage
+    dockerFileContextPath: ./
+    registry: cdpxlinux.azurecr.io
```

Comment:
Handcoded?

Reply:
Sorry, what is the question here?  Is the concern that `Dockerfile.aro-multistage` is hardcoded here?

Reply:
Registry name, should this not be variable, or parameter?

Reply:
The registry here doesn't matter for our purposes, though we still need to provide a value and of which we left it at the default value.  We update the tag and registry on the Ev2 side depending on the environment we're deploying to (int, prod, etc.)

The workflow, in short, is to build the docker image and publish it as a build artifact.  Ev2 will then consume, retag, and push to the appropriate registries.

https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/4601/Build-Docker-Images